lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -132,6 +136,8 @@ def instructor_dashboard_2(request, course_id):
     reports_enabled = configuration_helpers.get_value('SHOW_ECOMMERCE_REPORTS', False)
 
     sections = []
+    if course_id == 'course-v1:HarvardX+CS50+X':
+        log.info('Investigating log at %s : before _section_student_admin', time.time() - start_time)
```

Comment:
why are we logging the time here?

Reply:
`_section_student_admin` actually further calls the method `_is_small_course` that I actually changed so that's why I've put this here so I can check the time it took after that.

Reply:
Removing it. 👍